src/05-derivatives/02-common-data-types.md

```diff
@@ -0,0 +1,311 @@
+# Common data types
+
+## Processed, coregistered and/or resampled volumes
+
+Template:
+
+```Text
+<pipeline_name>/
+    sub-<participant_label>/
+        func|anat|dwi/
+        <source_keywords>[_space-<space>][_desc-<label>]_<suffix>.<ext>
+```
+
+Processing in this context means transformations of data that does not change
+the number of dimensions of the input and are not explicitly covered by other
+data types in the specification. Examples:
+
+-   Motion-corrected, temporally denoised, and transformed to MNI space bold
+    files.
+
+-   Inhomogeneity corrected and skull stripped T1w files.
+
+-   Motion-corrected DWI files.
+
+The `space` keyword is recomended to distinguish files with different underlying
+coordinate systems or registered to different reference maps. The
+`desc` keyword is a general purpose field with freeform values. To distinguish
+between multiple different versions of processing for the same input data the
+`desc` keyword should be used. Note that even though `space` and `desc` are
+optional at least one of them needs to be defined to avoid name conflict with
+the raw file.
+
+Examples:
+
+```Text
+pipeline1/
+    sub-001/
+        func/
+            sub-001_task-rest_run-1_space-MNI305_bold.nii.gz
+            sub-001_task-rest_run-1_space-MNI305_bold.json
+```
+
+```Text
+pipeline1/
+    sub-001/
+        func/
+            sub-001_task-rest_run-1_desc-MC_bold.nii.gz
+            sub-001_task-rest_run-1_desc-MC_bold.json
+```
+
+```Text
+pipeline1/
+    sub-001/
+        func/
+            sub-001_task-rest_run-1_desc-fmriprep_bold.nii.gz
+            sub-001_task-rest_run-1_desc-fmriprep_bold.json
+```
+
+All REQUIRED metadata fields coming from a derivative file’s source file(s) MUST
+be propagated to the JSON description of the derivative unless the processing
+makes them invalid (e.g., if a source 4D image is averaged to create a single
+static volume, a SamplingFrequency property would no longer be relevant). In
+addition, all processed files include the following metadata JSON fields:
+
+| **Key name**  | **Description**                                                                                 |
+| ------------- | ----------------------------------------------------------------------------------------------- |
+| SkullStripped | REQUIRED. Boolean. Whether the volume was skull stripped (non-brain voxels set to zero) or not. |
```

Comment:
This cannot be a required Key Name for an EEG derivatives dataset. It can be required for MR specific derivatives, but in that case it should not be in the "common data types" section (since MEG, EEG, iEEG and Behavior also fall under the common data types).